apps/modernization-ui/src/apps/search/layout/result/header/options/table/search-results-table-options.module.scss

```diff
@@ -5,7 +5,7 @@
 }
 
 .gear {
-    fill: colors.$primary;
+    fill: currentColor;
```

Comment:
Question: camel case works? docs seem to indicate `currentcolor`

Reply:
Yes, verified it in Firefox and Chrome

Reply:
[Formal syntax](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#formal_syntax) is camel case, the [section](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword) describing it is all lowercase